specs/core/0_beacon-chain.md

```diff
@@ -528,55 +488,45 @@ class BeaconBlock(Container):
 
 ```python
 class BeaconState(Container):
-    # Misc
-    slot: uint64
+    # Versioning
     genesis_time: uint64
-    fork: Fork  # For versioning hard forks
-    # Validator registry
-    validator_registry: List[Validator]
-    balances: List[uint64]
-    # Randomness and committees
-    latest_randao_mixes: Vector[Bytes32, LATEST_RANDAO_MIXES_LENGTH]
-    latest_start_shard: uint64
-    # Finality
+    slot: Slot
+    fork: Fork
+    # History
+    parent_block_header: BeaconBlockHeader
```

Comment:
Not sure about the semantics here.
The `latest_block_header` seen _should_ be the `parent` of the next block, but it is not a parent when recorded and not during any skipped slots when it is recorded into `block_roots`.

Reply:
Agreed that naming is awkward here. Brainstorming:

* `block_header` (matches the type nicely; vagueness hides difficult-to-explain subtleties)
* `partial_block_header` (a bit more accurate)
* `latest_block_header` (make an exception here for "latest" prefix)
* `latest_partial_block_header` (a bit of mouthful)

Reply:
`latest_` seems reasonable and purposefully descriptive here (as opposed to it just being thrown on all the history arrays).

Reply:
bump @JustinDrake 

Just handled merge conflict. let's resolve this and get merged.